### ISSUE TITLE
[search-bar][gmdl] query is always undefined

### ISSUE THREAD

**Issue by durandx** [KleeGroup/focus-components#313]
Since the last update of gmdl (focusjs 0.6.1), the broadcastQueryChange always broadcast a undefined value

https://github.com/KleeGroup/focus-components/blob/38bbf6c31a16397b57543bafc31342417e2e6fc5/src/search/search-bar/index.js#L101

Here : 
```
React.findDOMNode(this.refs.query)
```
evals as : 
```html
<div class="mdl-textfield mdl-js-textfield is-upgraded is-dirty is-focused" data-focus="input-text" data-reactid=".0.3.0.1.0.0.1.0" data-upgraded=",MaterialTextfield"><input class="mdl-textfield__input" placeholder="Rechercher" type="text" data-reactid=".0.3.0.1.0.0.1.0.0"><label class="mdl-textfield__label" data-reactid=".0.3.0.1.0.0.1.0.1"></label></div>
```

**Comment by pierr** [comment]
According to me this was fixed in the PR #315 
You can get it by downloading the latest patch version [v0.6.1-1](https://www.npmjs.com/package/focusjs-components)

**Comment by pierr** [comment]
Please reopen the issue if it is not fixed

**Comment by durandx** [comment]
I created the bug before going to see you directly. Now it's fixed in the v0.6.1-1 version

**Comment by Bernardstanislas** [comment]
:+1:
